_posts/2020-03-15-Sharezone-In-Review-March-Part-1.md

```diff
@@ -0,0 +1,88 @@
+Nebenbei haben wir an einer Zukunftsidee von Sharezone als Tool im Unterricht gebaut.
+
+*In "[Sharezone In Review]({{ site.baseurl }}/Introducing-Sharezone-In-Review)" stellen wir euch vor, was bei uns gerade bei Sharezone abläuft.\
+In dem Beitrag geht es um die ersten beiden Wochen im März.*
+
+## Öffentlichkeitsarbeit
+![]({{ site.baseurl }}/images/Sharezone-In-Review-March+-Part-1/sz-logo.svg)
+
+Bei dem Berg an Aufgaben fällt es uns oft schwer viel Öffentlichkeitsarbeit für Sharezone zu machen. Um so besser, dass Nils in den letzten zwei Wochen aber so richtig reingehauen hat: Von einem Vortrag auf einem Bundeskongress, einem Radio-Beitrag mit zugehörigem Artikel, zu einem Podcast, hat er viel von sich in den letzten zwei Wochen hören und blicken lassen.  
+
+### Vortrag "Warum jede:r ein Startup gründen sollte"
+Nils konnte vor einer versammelten Gruppe von Lehrkräften den Vortrag "Warum jede:r ein Startup gründen sollte". Als Beispiel ging es natülich um das beste Startup der Welt: Sharezone 😉
+
+Doch warum sollte denn jetzt überhaupt jeder ein Startup gründen? Kurzgefasst: Man lernt Gebietsübergreifend (Marketing, Informatik, Vertrieb, Design, Jura, etc.) sehr viele Sachen auf einer Art und Weise, die traditioneller Unterricht nicht vermittelt.    
+
+Der Vortrag wurde im Rahmen der 112. MNU-Bundeskonferenz zur Förderung des MINT-Unterrichts gehalten. Der Vortrag ist zwar (nach unserem Wissen) nicht auf Youtube, eine ältere Version des Vortrags von Nils kann sich aber [hier](https://www.youtube.com/watch?v=Bg0okdOOnZw) angeguckt werden.
+
+### Radio-Beitrag mit Artikel über Pilotschule in Berlin
+![Überschrift und Sub-Überschrift des rbb24-Artikels]({{ site.baseurl }}/images/Sharezone-In-Review-March+-Part-1/rbb-artice-intro.png)
+
+Neben dem Vortrag gabs noch mehr von Nils, diesmal im Doppelpakt. rbb24 veröffentlichte ein Radiosegment und [einen kurzen Artikel](https://www.rbb24.de/panorama/thema/corona/beitraege/2021/03/lern-app-berlin-pilotprojekt-grundschule.html) über eine Berliner Grundschule, welche Teil unseres Pilotschul-Programms ist.  
+
+Mit dem Pilotschul-Programm haben Schulen die Möglichkeit mit uns zusammen Sharezone für die ganze Schule einzurichten und konstenlos zu betreiben.   
+Dafür versuchen die Schulen möglichst viel Feedback für uns zu sammeln, damit wir wissen an welchen Stellen es in Sharezone für Schulen noch am meisten hakt.    
+
+Im dem Artikel wurde kurz angeschnitten was Sharezone ist und welche Vorteile es aus der Sicht des stellvertretenden Schulleiters bringt.  
+
+### Podcast über Flutter  
+![Bild von Nils mit Podcast-Thema "Flutter" von Podcast programmier.bar]({{ site.baseurl }}/images/Sharezone-In-Review-March+-Part-1/programmier.bar nils.png)
+
+Als letzten Punkt im PR-Marathon hat unser Spitzensporter Nils auch noch ein Podcast aufgenommen. Diesmal ging diesmal nicht um Sharezone, sondern um ein Werkzeug, welches wir benutzen: Flutter.
```

Comment:
Strenggenommen war ich sogar bei zwei Podcast 😉 Das ist der 2. Podcast: [digitacheles](https://www.digitacheles.com/)

Die Folge ist aber noch nicht online (geplant für nächste Woche).

In der Folge ging es um meinen Beitrag aus dem Spiegel-Bestseller "[Zukunftsrepublik](https://amzn.to/3vsJIDO)", wie das Lernen in der Zukunft aussehen sollte.

Bild: 
![image](https://user-images.githubusercontent.com/24459435/111209275-1bbb1480-85cc-11eb-9338-860a1006a504.png)

Kannst selber entscheiden, ob du es noch dazupacken möchtest.

Reply:
Nee zu Faul, dann lass für nächstes mal das reinnehmen :D Dan kann man den auch direkt verlinken.